alws/models.py

```diff
@@ -685,21 +694,52 @@ def __repr__(self):
 )
 
 
-class User(Base):
+class UserOauthAccount(SQLAlchemyBaseOAuthAccountTable[int], Base):
+    __tablename__ = 'user_oauth_accounts'
+
+    id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+
+    @declared_attr
+    def user_id(cls):
+        return sqlalchemy.Column(
+            sqlalchemy.Integer,
+            sqlalchemy.ForeignKey("users.id", ondelete="cascade"),
+            nullable=False
+        )
```

Comment:
I'm not sure but may be we should add `relationship` field for `User` model
or this field already inherited from `SQLAlchemyBaseOAuthAccountTable`?

Reply:
This field is required in base class, so since we're using a different table name I needed to patch foreignkey. And yes, relationship is already defined in base class